libs/utils/utils.js

```diff
@@ -424,6 +424,27 @@ export function decorateSVG(a) {
   }
 }
 
+function hasUrlWithPipe(str) {
+  const regex = /^https?:\/\/.*\|.*/;
```

Comment:
to be "cleaner" here i'd consider using the "non pipe" range in 1+ instead of any character so something like `/^https?[^\|]+\|.*/`

Reply:
the hasUrlWithPipe() func is no longer needed w/ the above `'img[alt*="|"]'` suggestion. 👍